golang/cmd/factoryinsight/http.go

```diff
@@ -50,7 +55,8 @@ func SetupRestAPI(accounts gin.Accounts, version string) {
 
 func handleInternalServerError(c *gin.Context, err error) {
 
-	zap.S().Errorw("Internal server error",
```

Comment:
Can we move all these reformatting stuff into a separate commit? it kind of blows this while PR

Reply:
@Scarjit you are still working in this?

Reply:
@JeremyTheocharis yes, working on this now, was waiting for my large codechange (the one with the new pre-push to be merged, cause it resolves this issue)